### PR TITLE
Documentation builder: fix relative paths for source code examples

### PR DESCRIPTION
**Pull request by anticrisis**
This pull request adjusts the relative paths for the source code examples in the User guide so that they are properly included in the resulting html. See https://github.com/libuv/libuv/issues/1427

### CODE REVIEW

File: docs/src/guide/basics.rst [hunk 1]
```diff
@@ -136,7 +136,7 @@ Handles are opaque structs named as ``uv_TYPE_t`` where type signifies what the
 handle is used for. 
 
 .. rubric:: libuv watchers
-.. literalinclude:: ../libuv/include/uv.h
+.. literalinclude:: ../../../../libuv/include/uv.h
```
Comment (saghul) [inline docs/src/guide/basics.rst]: this assumes the directory is called "libuv" but that need not be the case. doesn't `../../../include/uv.h` work?

Comment (cjihrig) [inline docs/src/guide/basics.rst]: Looks like just an artifact of how https://github.com/nikhilm/uvbook is structured.

**Comment by anticrisis** [comment]
Sorry, I should have thought more carefully about the relative paths. There is still one warning during the build, a reference to an undefined label, but addressing that would require a change to the content of the guide, which I am not at this point qualified to make.

```
/home/anticrisis/dev/libuv/docs/src/guide/utilities.rst:191: WARNING: undefined label: libuv-work-queue (if the link has no caption the label must precede a section header)
```

File: docs/src/guide/eventloops.rst [hunk 2]
```diff
@@ -19,7 +19,7 @@ these things can be a bit difficult to understand, so let's look at
 ``uv_run()`` where all the control flow occurs.
 
 .. rubric:: src/unix/core.c - uv_run
-.. literalinclude:: ../libuv/src/unix/core.c
+.. literalinclude:: ../../../../libuv/src/unix/core.c
```
Comment (saghul) [inline docs/src/guide/eventloops.rst]: same here, don't go all the way out

**Comment by saghul** [comment]
No worries, thanks! I left one last comment. While you are at it, can you please squash your commits and make the commit log conform to the guidelines in CONTRIBUTING.md? Thanks!

**Comment by anticrisis** [comment]
Ok, corrected the last reference to 'libuv' in relative paths, and I hope I did the squash thing right. Using --force on any command always makes me nervous.

**Comment by saghul** [comment]
@anticrisis Yep, you got it right, thanks!

**Comment by saghul** [comment]
Landed in cf38297b, thanks!

I see this is your first libuv contribution, welcome aboard! 🚢

**Pull request closed without merging**